api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -47,6 +51,12 @@
   private boolean disabled;
   private EmailVerificationStatus emailVerificationStatus;
 
+  @OrderColumn(name="PRINT_ORDER")
```

Comment:
What is "print_order"? Does this actually work?

I'm pretty sure the annotations for columns either need to go on fields or on getters, not both... so this probably belongs on getInstitutionalAffiliationSet()?

Reply:
Print order was actually wrong there, fixed. Will push up shortly.